clients/fossid-webapp/src/main/kotlin/model/status/ScanDescription.kt

```diff
@@ -19,8 +19,10 @@
 
 package org.ossreviewtoolkit.clients.fossid.model.status
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
```

Comment:
Commit message: A short explanation *why* this is necessary *now* would have been nice, like FossID version XY introduced new properties that are not needed right now, and to guard against this happening in the future.